src/NServiceBus.Transport.Msmq/MessagePump.cs

```diff
@@ -202,10 +202,19 @@ bool QueueIsTransactional()
             {
                 return inputQueue.Transactional;
             }
+            catch (MessageQueueException msmqEx)
+            {
+                var error = $"There is a problem with the input inputQueue: {inputQueue.Path}. See the enclosed exception for details.";
+                if (msmqEx.MessageQueueErrorCode == MessageQueueErrorCode.QueueNotFound)
+                {
+                    error = $"The inputQueue: {inputQueue.Path} does not exist. Run the queue creation PowerShell scripts that come packaged with the Nuget package, or explicitly call EnableInstaller() API to create the queues on startup.";
```

Comment:
>Run the queue creation PowerShell scripts that come packaged with the Nuget package

How about:

Run the `/Scripts/CreateQueues.ps1` script included in the `NServiceBus.Transports.Msmq` NuGet package.